n2y/simplify.py

```diff
@@ -1,8 +1,10 @@
 """
 Simplify data that has been returned from the Notion API.
 """
+import logging
 import re
-import sys
+
+logger = logging.getLogger('n2y.simplify')
```

Comment:
Usually people will just use `logger = logging.getLogger(__name__)`, since this will get the module name automatically.